src/contracts/pods/EigenPod.sol

```diff
@@ -443,21 +466,27 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         external
         onlyEigenPodManager
     {
-        // reduce the restakedExecutionLayerGwei
-        restakedExecutionLayerGwei -= uint64(amountWei / GWEI_TO_WEI);
-
```

Comment:
shouldn't we still be decrementing this? I am confused about the exact flow of these different variables (specifically the beaconChainETH shares and `restakedExecutionLayerGwei`) -- hoping some _high-level_ documentation should at least clarify the intended design